url.bs

```diff
@@ -2483,6 +2483,8 @@ interface URL {
            attribute USVString search;
   [SameObject] readonly attribute URLSearchParams searchParams;
            attribute USVString hash;
+
+  USVString toJSON();
```

Comment:
@annevk, yeah I'm okay with the PR in general. The bit about IDL notation is minor and does not affect the semantics anyway.